docs/BR.md

```diff
@@ -1595,10 +1589,22 @@ The issuing CA SHALL document in its Certificate Policy or Certification Practic
 
 ## 7.3 OCSP profile
 
+1. All OCSP responses MUST conform with RFC6960 and be the profile specified in RFC5019.
+2. Responses MUST be pre-produced as specified in RFC5019.
```

Comment:
@vanbroup I'm not sure how compromise of the delegated OCSP responder cert = compromise of the issuing CA. Surely people aren't giving `CA:TRUE` capabilities to those responders... are they?

Yeah, Timestamping is a complex beast, so I don't worry about that one, if only because it's not used in the Web PKI ;) I'm just trying to focus on the basic bits for OCSP.

To make sure I understand your suggestion:
* You'd like the requirement to pre-produce removed
  * You should only be able to do this if you're using Delegated OCSP Responder
  * Your Delegated OCSP Responder MUST be restricted in both validity period (does 30 days work?) and in capabilities (MUST NOT be a CA cert)
  * Restrictions on `id-pkix-ocsp-nonce` still apply
* Extend the validity period for `revoked` status to cover the entire certificate validity period
  * Is there any reason you can think, for the situation of the WebPKI, why revocation should not _also_ be extended to the beginning of the validity period (e.g. for a revoked response, `revoked.thisUpdate` = `cert.notBefore`, the `revoked.nextUpdate` = `cert.notAfter`, `producedAt` = revocation time)?


I'm not sure I understood your last part about response updates. Right now, the minimum is 8 hours (inherited from Microsoft) and the maximum is 7 days (also inherited from Microsoft). Updates must be produced at 1/2 the validity period, so if you had an OCSP response valid for 7 days, you'd only need to pre-produce a response once every 3.5 days. Does that meet your existing need?